5watchdog

```diff
@@ -134,11 +135,12 @@ fi
 # Initialize vars
 REBOOTRESET=0
 GPU_COUNT=$(nvidia-smi -i 0 --query-gpu=count --format=csv,noheader,nounits)
-COUNT=$((6 * GPU_COUNT))
+ACTIVE_GPU_COUNT = $(($GPU_COUNT - ${#DISABLED_GPU_ARRAY[@]}))
```

Comment:
I think line 138 should be:
ACTIVE_GPU_COUNT=$(($GPU_COUNT - ${#DISABLED_GPU_ARRAY[@]}))

Reply:
Just fixed the ACTIVE_GPU_COUNT, 
wonder if this is the cause of the GPU 0 not showing up issue.